### DISCUSSION TITLE
About status

### DISCUSSION THREAD

**Discussion by bepolymathe** [TechbeeAT/jtxBoard#577] [General]
Hi @patrickunterwegs 

Thank you for this project and this great application. I'm a long time Evolution and Thunderbird user and I'm a bit confused by the choice of "status" entries. 

In my understanding and use for many years, the tasks have the following status: 
- Unspecified: It is a normal task
- Waiting for an action: it is a task that depends on something to be done before: waiting for someone's answer or for another one to be finished. 
- In progress: a task that is started not finished. Often applies to long tasks. 

In Evolution it gives: 
- Not started
- Requires action
- In progress

In Jtx:
- Not specified
- To do
- In progress

I find that the "requires action" (2nd) status is a bit confusing between the three applications. In Jtx "To do" doesn't seem explicit enough... or do you have another understanding of this status? 🤔 
But maybe it's a translation problem... I work in French.

**Comment by patrickunterwegs** [comment]
Hey @bepolymathe ,
The problem here is that the definition of the iCalendar standard is not 100% clear:
https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.11

On the one hand the given status fields are defined. It is also defined that the status can be missing. But it is not clear if a missing status automatically means the first status. So there have been discussions on this and after all I decided to show a missing status as such.
But after all also other applications that don't get an explicit status would have to deal with it somehow and show them with a default status. 
So it's a bit hard to find the right compromise here.
In the upcoming version I have added the option to define custom statuses that map to a standard status. Maybe this will help to solve this problem as everybody can just make their individual status.

**Reply by patrickunterwegs** [reply]
Hey @bepolymathe again!
Sorry, I have missed a detail in your message and just reviewed it. 
You're right, in English the status is called "needs action". In French it says "À faire", so obviously "To do".

Sure we can correct it to a better translation! You can either do it directly (through the crowdin platform, see Website) or you just tell me what you think fits best.
I can communicate in french but I'm already my own translation would end up in something confusing 😅

**Reply by bepolymathe** [reply]
Hi, 

Thanks for reply. I completely follow your reasoning. I think the main issue is to keep it simple for the users who need it and more precise for those who need a more advanced use. So it seems to me that any default task should be understood as a "to do" task. If a status is specified it means that it has something special. So the translation of the 2nd status must ce "requires an action" ("Nécessite une action" in french) seems to me more in line with what other applications do.

The fact of being able to offer a specific status in the application is very interesting for those who only use the Phone application. But it will continue to be a problem until we have a desktop application.

Thanks for your efforts 😉

**Reply by patrickunterwegs** [reply]
Merci, alors j'ai corrigé la traduction, j'espère que personne d'autre va se plaindre après 😅
Mais bon, j'entends que c'est mieux comme ça.
Si ça t'intéresse de faire des autres traductions, tu peux joindre le projet ici:

https://crowdin.com/project/jtx-board/invite?h=a8fd45e2dfea25534eda503b441476ea1545967

**Reply by bepolymathe** [reply]
Merci. Je vais m'inscrire pour la traduction as soon as possible 😉

**Reply by bepolymathe** [reply]
Edit: fait/done
